Sources/NIOCore/ByteBuffer-multi-int.swift

```diff
@@ -0,0 +1,2393 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2021 Apple Inc. and the SwiftNIO project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+/// NOTE: THIS FILE IS AUTO-GENERATED BY THE SCRIPT FOUND AT THE END OF THE FILE
+
+extension ByteBuffer {
+    @inlinable
+    @_alwaysEmitIntoClient
```

Comment:
I added the `@_alwaysEmitIntoClient` now because otherwise we'll add A LOT of binary size for the unused generic versions of this. We only want to have the specialised ones.